packages/@okta/vuepress-site/docs/guides/anything-as-a-source/index.md

```diff
@@ -0,0 +1,407 @@
+
+* You can only load data to an Identity Source Session when it’s in the `CREATED` status.
+* There can only be one Identity Source Session in the `CREATED` status for an identity source.
+* An Identity Source Session with the `CREATED` or `TRIGGERED` status is considered active.
+* If there are no API requests in 24 hours for an Identity Source Session that has the `CREATED` status, then the status is set to `EXPIRED` and the session can no longer be used.
+* You can’t process an Identity Source Session in parallel for the same identity source. If you trigger a session for an identity source that already has a session processing, then the newly triggered session is queued until the current import process completes.
+* You can't create a new Identity Source Session in less than five minutes after triggering an active session associated with the same identity source. If Okta receives a new Identity Source Session request within five minutes of an active Identity Source Session with the `CREATED` or the `TRIGGERED` status, Okta returns a 400 Bad Request response.
+
+> **Note:** You can use the [List active Identity Source Sessions](/docs/reference/api/xaas/#list-active-identity-source-sessions) request to return active Identity Source Sessions for an identity source.
+
+### Bulk-load requests
+
+There are two types of bulk-load requests:
+
+* `/bulk-upsert`: Insert or update user profiles in the bulk-load request
+* `/bulk-delete`: Deactivate the user profiles in the bulk-load request
+
+You can load up to 200 KB of data in a single bulk-load (`/bulk-upsert` or `/bulk-delete`) request for an Identity Source Session. This equates to 200 user profiles. To load more user profiles, you can make multiple bulk-load requests to the same session. The maximum number of bulk-load requests for a session is 50. If you exhaust the maximum number of bulk-load requests and you still need to load more user profiles, then create another Identity Source Session object for the additional user profiles. Keep in mind that you can only load user data to an Identity Source Session with the `CREATED` status.
+
+> **Note:** Only `"importType": "INCREMENTAL"` is currently supported for an Identity Source Session.
+
+### Bulk user profile data
+
+The bulk-load request contains an array of external [Identity Source User Profile For Upsert](/docs/reference/api/xaas/#identity-source-user-profile-for-upsert-object) or [Identity Source User Profile For Delete](/docs/reference/api/xaas/#identity-source-user-profile-for-upsert-object) objects that contain the following:
+
+* `externalId`: The unique identifier from the HR source and is assumed to be immutable (never updated for a specific user). This value is used as a key to determine if a new user needs to be created or if an existing user needs to be updated.
```

Comment:
This is something the system does (Okta). If Okta finds the same key in its source, it will do an update. If Okta doesn't find the externalId key, it will add a new user profile.